### ISSUE TITLE
How to get topics, probability after loading saved Topic Model

### ISSUE THREAD

**Issue by TAsUjxnMIL** [MaartenGr/BERTopic#417]
Hello Maarten,
is there a possibility get the topics and the probability after loading the saved Topic Model? 

Note: 
I am refering to those two return values
topics, probs = topic_model.fit_transform(docs)

Or do I have to run a second time this instruction with the loaded topic_model?

Thank you for your help!

**Comment by KnobKraus** [comment]
You should call topic_model.transform(docs) (not fit_transform) , to reproduce your results. Be aware that due to UMAPs stochastic nature, your results will be close to, but not exactly the same as the probs you calculated in the previous fit_transform run. If you have bad luck, your results may even differ a lot from previous runs. To avoid this, you should always, if you plan to save your model and load it again for later use, set a random_state for the UMAP algorithm, to avoid changing results on each transform call.

**Comment by MaartenGr** [comment]
@TAsUjxnMIL After you have saved and loaded your model you can indeed try to use `.transform()` again. However, due to the stochastic nature of UMAP this might result in different values. Moreover, setting a `random_state` is likely to slow down UMAP which might not be preferred but does result in stable results. 

You can also try the following:
```python
probs = topic_model.hdbscan_model.probabilities_
topics = topic_model._map_predictions(topic_model.hdbscan_model.labels_)
```

Fortunately, the labels and probabilities of the most recent fit are saved in the HDBSCAN model from which they can be extracted.
